pyfolio/tears.py

```diff
@@ -352,6 +358,18 @@ def create_position_tear_sheet(returns, positions, gross_lev=None,
 
     plotting.plot_holdings(returns, positions_alloc, ax=ax_holdings)
 
+    if sector_mappings is not None:
+        sector_exposures = pos.get_sector_exposures(positions, sector_mappings)
+
+        if sector_exposures is None:
```

Comment:
I don't see a case where `sector_exposures` could return `None`.  What's the intent behind this check?